packages/sit-onyx/src/components/OnyxRadioButtonGroup/OnyxRadioButtonGroup.vue

```diff
@@ -1,15 +1,12 @@
-<script lang="ts" setup generic="TValue extends SelectionOptionValue">
-import type { TargetEvent } from "../../types/dom";
+<script lang="ts" setup generic="TValue extends SelectOptionValue">
```

Comment:
Use SelectOption for modelValue everywhere, as discussed
```suggestion
<script lang="ts" setup generic="TValue extends SelectOption">
```

Reply:
Lets discuss this with the other devs in a follow up task #951